src/Digdir.Domain.Dialogporten.Infrastructure/Digdir.Domain.Dialogporten.Infrastructure.csproj

```diff
@@ -1,12 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <ItemGroup>
+        <PackageReference Include="AspNetCore.HealthChecks.NpgSql" Version="8.0.2" />
```

Comment:
Hvorfor hører dette hjemme i infrastruktur? Dette er et presentasjonslag ansvar som ikke nødvendigvis gjelder for alle presentasjonslag IMO